aioxmpp/nonza.py

```diff
@@ -752,19 +756,16 @@ class SMFailed(SMXSO):
     """
     TAG = (namespaces.stream_management, "failed")
 
-    condition = xso.ChildTag(
-        tags=stanza.STANZA_ERROR_TAGS,
-        default_ns=namespaces.stanzas,
-        allow_none=False,
-        declare_prefix=None,
+    condition = xso.Child(
+        [member.xso_class for member in errors.ErrorCondition],
+        required=True,
     )
 
     def __init__(self,
-                 condition=(namespaces.stanzas, "undefined-condition"),
```

Comment:
Here the condition changed (undefined-condition => item-not-found). Is this intentional?